src/boot_O2_g3/fault_drawer.c

```diff
@@ -16,7 +18,7 @@ FaultDrawer sFaultDrawerDefault = {
     GPACK_RGBA5551(0, 0, 0, 0),         // backColor
     22,                                 // cursorX
     16,                                 // cursorY
-    (u32*)sFaultDrawerFont,             // font
+    sFaultDrawerFont,                   // font
     8,                                  // charW
     8,                                  // charH
     0,                                  // charWPad
```

Comment:
Could you label the colours too? Most likely they're the same as the [ASCII terminal ones](https://en.wikipedia.org/wiki/ANSI_escape_code#Colors), i.e.
Black
Red
Green
Yellow
Blue
Magenta
Cyan
White
and two greys.